openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1168,9 +1170,19 @@ A Credential Issuer is identified by a case sensitive URL using the `https` sche
 
 The Credential Issuer's configuration can be retrieved using the Credential Issuer Identifier.
 
-Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`. 
+Credential Issuers publishing metadata MUST make a JSON document available at the path formed by concatenating the string `/.well-known/openid-credential-issuer` to the Credential Issuer Identifier. If the Credential Issuer value contains a path component, any terminating `/` MUST be removed before appending `/.well-known/openid-credential-issuer`.
+
+To fetch the Credential Issuer Metadata, a requester MUST send a HTTP GET to the path formed following the steps above utilizing TLS. The Credential Issuer MUST return a JSON document compliant with this specification using the `application/json` media type and a HTTP Status Code 200.
+
+The Wallet is RECOMMENDED to send an Accept-Language Header in the HTTP GET request to indicate the particular languages he is interested in. The Credential Issuer is RECOMMENDED to send a subset of internationalized display data containing the requested languages and using the HTTP Content-Language Header. The Credential Issuer MAY ignore the Accept-Language Header and send all supported languages or a any chosen default subset.
+
+Below is a non-normative example of a Batch Credential Request:
 
-The path formed following the steps above MUST point to a JSON document compliant with this specification. The document MUST be returned using the `application/json` media type.
+```
+GET /.well-known/openid-credential-issuer HTTP/1.1
+Host: server.example.com
+Accept-Language: de-DE
```

Comment:
I believe it's permitted to have more complex values, so I think we may want an example that's more complex.

```suggestion
Accept-Language: fr-CH, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5
```

Reply:
I agree that a more complex example than 1 language makes sense, but I am not sure the weighted example is a good fit here or rather might be confusing. Instead of using a weighted request, wouldn't we instead just provide the list of languages we are interested in and the response would include all the fitting languages supported by the issuer?

If I understand weighted accept-language usage correctly, it is supposed to indicate a favorite language and the response is the best fit the server could find (in one language).

Reply:
I agree, we should make clear what the weighting means for us.

Reply:
discussed on the WG call. agreed that wallet can use weighted multiple languages, but hte issuer is free to return whatever, even if it is only one of the requested languages. @paulbastian to make sure this is reflected in the spec text. good to merge after that.